parser/prism/Translator.cc

```diff
@@ -933,6 +933,11 @@ unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
 
             return translate(reinterpret_cast<pm_node *>(programNode->statements));
         }
+        case PM_POST_EXECUTION_NODE: {
+            auto postExecutionNode = reinterpret_cast<pm_post_execution_node *>(node);
+            auto body = translate(reinterpret_cast<pm_node *>(postExecutionNode->statements));
```

Comment:
Should this just call `translateStatements` directly?

```suggestion
            auto inlineIfSingle = true;
            auto body = translateStatements(postExecutionNode->statements, inlineIfSingle);
```